Source/Gateway/Concepts/Logins/Logo.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Dolittle.Concepts;
+
+namespace Concepts.Logins
+{
+    public class Logo : ConceptAs<byte[]>
```

Comment:
Logos need to be dealt with in a different way. We will have a location where tenants information lives with logos and all and this will be mounted as a volume. By convention from the `TenantId` and when we have the `TenantStorage` from @jakhog, we can just have a file called `logo.png`. So we don't need this as a concept. 
We just need to come up with a way to deal with uploading to this folder. I've added an issue for this (
https://github.com/dolittle-platform/Sentry/issues/57).

Reply:
Btw: Another thing; concepts of `byte[]` feels a bit icky. A Concept is something that represents a domain concept and is more often a thing that has a reference to something. So if you were to take the Logo as an example, it would be the identifier of a Logo - the thing that uniquely identified it - while its binaries would be on disk somewhere and accessible by providing the identity in the form of a Logo to whatever sub-system was responsible for Logo management.